recipes/compss/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "pycompss" %}
+{% set version = "2.4" %}
+{% set sha256 = "237a8ed3842a6d836a330f4c707cd12781ef6cfad58668d52a84be286384a0ba" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #url: http://compss.bsc.es/repo/sc/stable/COMPSs_{{ version }}.tar.gz
+  url: http://bscgrid05.bsc.es/~lezzi/COMPSs_{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:  
+  number: 0
+  skip: True  # [osx]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - autoconf
+    - libtool
+    - automake
+    - autoconf
+    - make
+    - pkg-config
+    - bison
+    - flex
+    - libtool
+  host:
+    - gmp
+    - pip
+    - setuptools
+    - python >=3
+    - boost
+    - openjdk=8
```

Some parts of conda-forge need a space here:
```suggestion
    - openjdk =8
```